package.json

```diff
@@ -57,5 +44,13 @@
     "packageName": "int_algolia_sfra",
     "paths": {
         "base": "../storefront-reference-architecture/cartridges/app_storefront_base/"
+    },
+    "dependencies": {
+        "@algolia/autocomplete-theme-classic": "^1.8.3",
```

Comment:
I'd use pinned versions to avoid any surprise

Reply:
switched to exact version numbers